quinn-proto/src/connection/mod.rs

```diff
@@ -901,14 +909,14 @@ impl Connection {
 
         Some(Transmit {
             destination: self.path.remote,
-            contents: buf.freeze(),
+            size: buf.len(),
             ecn: if self.path.sending_ecn {
                 Some(EcnCodepoint::Ect0)
             } else {
                 None
             },
             segment_size: match num_datagrams {
-                1 => None,
+                1 | 0 => None,
```

Comment:
Why is this necessary? It doesn't obviously make sense to me.

Reply:
No it is not.
I had an issue running the tests while doing the first iteration.
Thanks for the catch 🙏